waku/v2/node/wakunode2.nim

```diff
@@ -354,19 +357,8 @@ proc lightpush*(node: WakuNode, topic: Topic, message: WakuMessage): Future[Waku
   let rpc = PushRequest(pubSubTopic: topic, message: message)
   return await node.wakuLightPush.request(rpc)
 
-proc lightpush*(node: WakuNode, topic: Topic, message: WakuMessage, handler: PushResponseHandler) {.async, gcsafe,
-  deprecated: "Use the no-callback version of this method".} =
-  ## Pushes a `WakuMessage` to a node which relays it further on PubSub topic.
-  ## Returns whether relaying was successful or not in `handler`.
-  ## `WakuMessage` should contain a `contentTopic` field for light node
-  ## functionality.
-
-  let rpc = PushRequest(pubSubTopic: topic, message: message)
-  let res = await node.wakuLightPush.request(rpc)
-  if res.isOk():
-    handler(res.value)
-  else:
-    error "Message lightpush failed", error=res.error()
+proc lightpush2*(node: WakuNode, topic: Topic, message: WakuMessage) {.async, gcsafe.} =
+  discard await node.lightpush(topic, message)
```

Comment:
I had to create this _fire-and-forget_ method because it is required by `chat2` example

Reply:
Wouldn't it be easier to just dispatch the future in `chat2` and discard it? `chat2` is a bit of a mess, so I don't think we should design around its idiosyncrasies.

Reply:
Indeed, `chat2` is a nightmare regarding this kind of PRs.

Here the problem is that if I remove the method from here, I need to wrap it in an anonymous proc and discard the result in 2 places in the `chat2` code:

* https://github.com/status-im/nwaku/blob/refactor-wakunode-remove-deprecated-methods/examples/v2/chat2.nim#L244
* https://github.com/status-im/nwaku/blob/refactor-wakunode-remove-deprecated-methods/examples/v2/chat2.nim#L273

Because we are spawning directly the `Future` returned by the waku node `lightpush()` proc 😞